tests/resources/rebuild_example_data

```diff
@@ -14,6 +15,8 @@ from zipline.testing import test_resource_path, tmp_dir
 from zipline.utils.cache import dataframe_cache
 
 
+matplotlib.use('Agg')
```

Comment:
It's required to rebuild the example data only because the example algos call an `analyze` function, which plots graphs of performance results; for some reason (most likely because of our recent update to the matplotlib version) a matplotlib window will open up and the `rebuild_example_data` script waits on the user to manually close that window (thus killing that process) before it continues.

By explicitly stating the `Agg` backend, this issue doesn't occur and no matplotlib windows show up when running the script.